cookbooks/aws-parallelcluster-platform/recipes/install/cuda.rb

```diff
@@ -50,7 +50,9 @@
   cwd '/tmp'
   code <<-CUDA
     set -e
-    ./cuda.run --silent --toolkit --samples
+    mkdir /cuda-install
+    ./cuda.run --silent --toolkit --samples --tmpdir=/cuda-install
```

Comment:
`/cuda-install` is meant to be a temporary directory, so we should create it under `/tmp`.
That said it's fine to use recursive deletion of that folder since it's a temp directory created by us and not meant ot be used by the user.

Reply:
`/tmp` on some OSes has size limit. That's why we had to change the directory

Reply:
Then please add a comment on top of it to explain why we are not using tmp dir.
However, we should be in control of the tmp dir size and adjust it to our needs.
May you please track in the backlog the possibility to control the /tmp size as a separate partition (that is also a storage best practice)?

Reply:
Done, added a comment